banzai/celery.py

```diff
@@ -86,10 +86,13 @@ def stack_calibrations(self, min_date: str, max_date: str, instrument_id: int, f
                                                                       min_date, max_date, include_bad_frames=True,
                                                                       db_address=runtime_context.db_address))
     expected_image_count = 0
-    for block in blocks:
-        for molecule in block['molecules']:
-            if frame_type.upper() == molecule['type']:
-                expected_image_count += molecule['exposure_count']
+    for observation in observations:
+        for configuration in observation['request']['configurations']:
+            if frame_type.upper() == configuration['type']:
+                for instrument_config in configuration['instrument_configs']:
+                    expected_image_count += instrument_config['exposure_count']
+    logger.info('expected image count: {0}'.format(str(expected_image_count)))
```

Comment:
Suuuuuuuper nitpicky but let's combine these into a single logging statement, because perusing logs is terrible when we're effectively using these two statements to just compare two numbers.

Reply:
Good call.